### PR TITLE
Try updating Danger to 8.x

### PR DESCRIPTION
**Pull request by dnkoutso**
This updates the gem `kramdown` to 2.3.x which resolves a high vulnerability issue in this repo.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -229,7 +254,7 @@ GEM
     netrc (0.11.0)
     no_proxy_fix (0.1.2)
     notify (0.5.2)
-    octokit (4.18.0)
+    octokit (4.22.0)
```
Comment (amorde) [inline Gemfile.lock:257]: when I did this earlier it broke Danger after merging. I think there's something that changed with the way auth works between these two versions https://github.com/CocoaPods/CocoaPods/pull/11225

Comment (dnkoutso) [reply Gemfile.lock:257]: ah let me fix that.

Comment (dnkoutso) [reply Gemfile.lock]: updated.

**Comment by orta** [comment]
Looks reasonable to me 👍🏻

**Pull request merged**